### PR TITLE
Added ionic SDK content(#2vjxcqb)

### PR DESCRIPTION
**Pull request by disha1202**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "order-routing-rules",
+  "version": "1.0.2",
```
Comment (ymaheshwari1) [inline package.json:3]: Change version
```suggestion
  "version": "0.0.1",
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# IonicSDK for Ionic 5.x/ Vue applications
+
+## How to use?
+
+- Clone your repository  
+`git clone <your repository url>`
+
+- Create a new branch
+- Add IonicSDK repo URL to remote
```
Comment (ymaheshwari1) [inline README.md:9]: Replace ionicSDK with Order Routing Rules

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (disha1202) [reply package.json]: Updated app version

File: README.md (hunk 2 above)
Comment (disha1202) [reply README.md]: Made the requested change

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HotWax Commerce</title>
```
Comment (ymaheshwari1) [inline public/index.html:5]: Change the title

File: src/services/UserService.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    method: "post",
+    data: {
+      'USERNAME': username, 
+      'PASSWORD': password
+    }
+  });
+}
+
+const getProfile = async (): Promise <any>  => {
+    return api({
```
Comment (ymaheshwari1) [inline src/services/UserService.ts:16]: Indentation issue.

File: src/router/index.ts [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import { createRouter, createWebHistory } from '@ionic/vue-router';
+import { RouteRecordRaw } from 'vue-router';
+import Home from '@/views/Home.vue'
+import Login from '@/views/Login.vue'
+import Settings from "@/views/Settings.vue"
+import store from '@/store'
+
+const authGuard = (to: any, from: any, next: any) => {
+  if (store.getters['user/isAuthenticated']) {
+      next()
```
Comment (ymaheshwari1) [inline src/router/index.ts:10]: Fix the indentation

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**